README.md

```diff
@@ -110,6 +110,7 @@ Thanks goes to these wonderful people ([emoji key](https://github.com/kentcdodds
 | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
+| [<img src="https://avatars3.githubusercontent.com/u/1233347?v=4" width="100px;"/><br /><sub>Alex Shelkovskiy</sub>](https://github.com/alexshelkov)<br />[🐛](https://github.com/rpearce/react-medium-image-zoom/issues?q=author%3Aalexshelkov "Bug reports") |
```

Comment:
@alexshelkov this is you being added to our all-contributors list. Thanks for your contribution to the project!